packages/plugin-e2e/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@grafana/plugin-e2e",
+  "version": "0.0.1",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "module": "./dist/esm/index.js",
+  "files": [
+    "./dist",
+    "./compiled",
+    "./package.json"
+  ],
+  "repository": {
+    "directory": "packages/plugin-e2e",
+    "url": "https://github.com/grafana/plugin-tools"
+  },
+  "author": "Grafana",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "registry": "https://registry.npmjs.org/",
+    "access": "public"
+  },
+  "scripts": {
+    "clean": "rm -rf ./dist ./compiled",
+    "build": "npm run clean && tsc -p ./tsconfig.json && rollup -c rollup.config.ts",
+    "build2": "tsc -p ./tsconfig.json && rollup -c rollup.config.ts",
```

Comment:
Is this for testing something?

Reply:
Oh this was just for debugging. Removed now.